src/client/Microsoft.Identity.Client/Internal/Logger/LegacyIdentityLoggerAdapter.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
+using Microsoft.Identity.Client.Core;
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.Internal.Logger
+{
+    internal class LegacyIdentityLoggerAdapter : ILoggerAdapter
+    {
+        LogLevel _minLogLevel = LogLevel.Always;
```

Comment:
Should this not be LogLevel.Error? Since it is the minLogLevel

Reply:
Minimum is always instead of error since it is the lowest one. Also, this value will always be set by the user. This default is really just for testing.